### ISSUE TITLE
Show what for users password is needed.

### ISSUE THREAD

**Issue by manu3l** [mate-desktop/mate-polkit#13]
As written in a downstream bug, https://bugs.mageia.org/show_bug.cgi?id=12145
mate-polkit doesn't show if the needed password is an users or the root one. Other polkit agent like the Gnome one show an avater.

A simple text could be enough.

Thanks

**Comment by NiceandGently** [comment]
@manu3l
Is this issue still exists with french translation?

**Comment by hickford** [comment]
Yes please to a clearer message! When I see the 'authentication is required' dialog I never know whether to type the root password or my own password (I use both `su` and `sudo` in the terminal).

> An application is attempting to perform an action that requires privileges. Authentication is required to perform this action. Password:

To me that suggests it wants the root password, but testing it now, it actually wants my user password. Ideally the message would be idiotproof in clarify:

> Authentication is required to perform this action. **Your password:**

or 

> Authentication is required to perform this action. **Password for user *tim*:**



![authentication-is-required](https://cloud.githubusercontent.com/assets/105314/14619768/085f617a-05b1-11e6-8fb6-697dfc3ae927.png)

**Comment by hickford** [comment]
Alternatively the message could mention the word 'sudo'. Anything that at a glance would tell me to type my user password rather than root's password.

**Comment by lukefromdc** [comment]
Question: how many people are using MATE that don't know the sudo password is their password and not the root password? My sister would never know the difference, this sort of thing gets me phone calls for support.

**Comment by hickford** [comment]
@lukefromdc Good point. Perhaps "Authentication is required to perform this action. **Your password**:" would be clearer

**Comment by lukefromdc** [comment]
This would work fine for most people, who by automatic habit would use their normal password
